build.gradle

```
@@ -237,6 +237,7 @@ allprojects {
             jvmArgs.add getJvmModulePath()
             jvmArgs.add '--add-modules'
             jvmArgs.add getJvmAdditionalModules()
+            excludeFilter = file("${project.rootDir}/spotbugs-exclude.xml")
```

An alternative way could be to evaluate the file from the rootProject.
```suggestion
            excludeFilter = rootProject.file('spotbugs-exclude.xml')
```